i18n/sv/cosmic-launcher.ftl

```diff
@@ -1 +1,2 @@
 app-name = Cosmic Programstartare
+type-to-search = Skriv för att söka efter program eller skriv "?" för mer alternativ...
```

Comment:
skriv "?" för fler alternativ. Sounds better imo.

Reply:
@williameliasson-dev alright changed

thanks for the help, tack för hjälpen